test/e2e/apimachinery/webhook.go

```diff
@@ -71,12 +71,20 @@ var serverWebhookVersion = utilversion.MustParseSemantic("v1.8.0")
 
 var _ = SIGDescribe("AdmissionWebhook", func() {
 	var context *certContext
+	var ns string
+	var c clientset.Interface
 	f := framework.NewDefaultFramework("webhook")
 	framework.AddCleanupAction(func() {
```

Comment:
Oh, this fix is still not complete. The CleanupAction is called after the entire suite is done, but we need to call these cleanups after each `It` block. @cheftako has some code in his PR that's doing the right thing I think.

Reply:
> we need to call these cleanups after each `It` block

Why not just use `AfterEach`?

> @cheftako has some code in his PR that's doing the right thing I think.

Would you provide the link to the PR?

Reply:
> Why not just use AfterEach?

That is what I meant :)

https://github.com/kubernetes/kubernetes/pull/55829

Reply:
Thanks for fixing the CleanupAction! I'll go fix the webhook specific tests, that test shouldn't use CleanupAction from the beginning.

Reply:
This PR is not trying to modify any e2e test, but just make AddCleanupAction work as intended (instead of noop).